markup5ever/interface/mod.rs

```diff
@@ -35,6 +60,66 @@ impl<'a> fmt::Debug for ExpandedName<'a> {
     }
 }
 
+// impl<'a> AsRef<ExpandedName<'a>> for ExpandedName<'a> {
+//     fn as_ref(&self) -> &ExpandedName<'a> {
+//         self
+//     }
+// }
+
+// /// A wrapper around an `ExpandedName` which allows it to be borrowed from a RefCell
+// pub struct ExpandedNameRef<'a>(pub Ref<'a, ExpandedName<'a>>);
+
+// impl<'a> AsRef<ExpandedName<'a>> for ExpandedNameRef<'a> {
+//     fn as_ref(&self) -> &ExpandedName<'a> {
```

Comment:
Out of curiosity, why is this code commented?

Reply:
This was a dead-end experiment (it ended up being impossible to actually implement AsRef in the relevant circumstances). The new approach with a custom trait is working for me. I'll go through and clean up later.

Reply:
Done!